src/Base.cpp

```diff
@@ -0,0 +1,74 @@
+#include "Base.h"
+#include "Marine.h"
+#include<random>
+
+Base::Base() {
+	this->setX((MAP_WIDTH/2)-BASE_WIDTH);
+	this->setY((MAP_HEIGHT/2)-BASE_HEIGHT);
```

Comment:
For parameters please try and keep parts spaced out as per the style guide
`this->setX((MAP_WIDTH/2)-BASE_WIDTH);`
should be
`this->setX((MAP_WIDTH / 2) - BASE_WIDTH);`